plugin/valgrind.vim

```diff
@@ -123,8 +123,8 @@ function! s:Valgrind( ... )
     else
         silent execute 'split '.l:tmpfile
     endif
-    silent execute 'g!/^==\d*==/d'
-    silent execute '%s/^==\d*== //e'
+    silent execute 'g!/^\(\d\+: \)\===\d*==/d'
```

Comment:
Either that or a hook that removes noise. For instance, I'm thinking of a possible encapsulation of each line by an XML node. In that case, we'll need to pre-process the lines differently.